packages/example-studio/migrations/renameField.js

```diff
@@ -0,0 +1,43 @@
+import baseClient from 'part:@sanity/base/client'
+
+// Remove when we get support for sanity exec --with-user-token
+import ConfigStore from 'configstore'
+const authToken = new ConfigStore('sanity', {}, {globalConfigPath: true}).get('authToken')
+const client = baseClient.config({token: authToken})
+// ---
+
+// Fetching documents that matches the precondition for the migration
+const fetchDocuments = () =>
+  client.fetch(`*[_type == 'author' && defined(fullname)] {_id, _rev, fullname}`)
```

Comment:
Should probably make it explicit how many items we are fetching (at most). _Might_ also be worth mentioning that `defined()` will return `false` for an empty array, which might not be obvious.